src/System.AppContext/tests/AppContext.Switch.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+public class AppContextSwitchesTests
+{
+    private static Random randomNumbers = new Random((int)DateTime.Now.Ticks);
+
+    [Fact]
+    public void SwitchNotFound()
+    {
+        string switchName = GetRandomSwitchName();
+        bool isEnabled;
+        Assert.False(AppContext.TryGetSwitch(switchName, out isEnabled));
```

Comment:
Consider adding an assert for the value of isEnabled as well.  I assume it should be Assert.False.